index.html

```diff
@@ -0,0 +1,133 @@
+      </section>
+      
+      <!-- Main Body -->
+      <section class = "body-container">
+        <section class = "content-section">
+          <div class = "content-card">
+
+            <img src = "image/hotitem.png">
+
+            <div class = "content-text">
+              <span class = "content-label">Hot item</span>
+            
+              <h2 class = "content-title">
+                인기 상품을 <br>
+                확인해보세요
+              </h2>
+            
+              <p class = "content-description">
+                가장 HOT한 중고거래 물품을 <br>
+                판다 마켓에서 확인해 보세요
+              </p>
+
+            </div>
+          </div>
+        </section>
+        
+        <section class = "content-section">
+          <div class = "content-card">
+
+            <div class = "content-text">
+              <span class = "content-label">Search</span>
+              <h2 class = "content-title">
+                구매를 원하는<br>
+                상품을 검색하세요
+              </h2>
+              
+              <p class = "content-description">
+                구매하고 싶은 물품은 검색해서 <br>
+                쉽게 찾아보세요
+              </p>
+            </div>
+            
+            <img src = "image/search.png">
+
+          </div>
+        </section>
+        
+        <section class = "content-section">
+          <div class = "content-card">
+
+            <img src = "image/register.png">
+            
+            <div class = "content-text">
+              <span class = "content-label">Register</span>
+              <h2 class = "content-title">
+                판매를 원하는 <br>
+                상품을 등록하세요
+              </h2>
+              
+              <p class = "content-description">
+                어떤 물건이든 판매하고 싶은 상품을 <br>
+                쉽게 등록하세요
+              </p>
+            </div>
+          </div>
+        </section>
+        
+        <!-- Main Footer -->
+        <section class = "content-footer">
+            <div class = "cf-content">
+              <h3 class = "cf-text">
+                믿을 수 있는 <br>
+                판다마켓 중고 거래
+              </h3>
+              <img class = "cf-img" src = "image/footer.png">
+            </div>  
+        </section>
+      </section>
+    </main>
+    
+    <!-- Footer -->
+    <footer>
+      <div>©codeit - 2024</div>
+      <div id="footer-content">
+        <a class = "footer-menu" href="privacy.html">Privacy Policy</a>
+        <a class = "footer-menu" href="faq.html">FAQ</a>
+      </div>
+      <div id="sns-link">
+        <a href="https://www.facebook.com/" target="_blank"><img src="image/facebook.png" alt="페이스북" width="20"/></a>
+        <a href="https://twitter.com/" target="_blank"><img src="image/twitter.png" alt="트위터" width="20"/></a>
+        <a href="https://www.youtube.com/" target="_blank"><img src="image/youtube.png" alt="유튜브" width="20"/></a>
+        <a href="https://www.instagram.com/" target="_blank"><img src="image/insta.png" alt="인스타그램" width="20"/></a>
+      </div>
```

Comment:
💊 제안
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
      <ul id="sns-link">
        <li>
          <a href="https://www.facebook.com/" target="_blank">
            <img src="image/facebook.png" alt="페이스북" width="20"/>
          </a>
        </li>
      </ul>
```